.github/workflows/build.yml

```diff
@@ -37,27 +37,27 @@ jobs:
                         symfony-version: ''
 
                     # MacOS on latest PHP only
-                    -   php: 8.1
+                    -   php: 8.2
                         os: macos-latest
                         composer-mode: update
                         symfony-version: ''
 
                     # Windows on latest PHP only
-                    -   php: 8.1
+                    -   php: 8.2
                         os: windows-latest
                         composer-mode: update
                         symfony-version: ''
 
                     # Symfony jobs:
-                    -   php: 8.1
+                    -   php: 8.2
```

Comment:
As Symfony 4.4 reaches EOM before the 8.2 release, I suggest keeping that job on PHP 8.1

Reply:
Aha, we should think about dropping it too then?